fundamentals/android/auth-SiWG/Notes/app/src/main/java/com/notes/app/screens/account_center/AccountCenterViewModel.kt

```diff
@@ -35,8 +44,27 @@ class AccountCenterViewModel @Inject constructor(
 
     fun onSignInClick(openScreen: (String) -> Unit) = openScreen(SIGN_IN_SCREEN)
 
-    fun onSignInWithGoogleClick(openScreen: (String) -> Unit) {
+    fun onGetCredentialResponse(result: GetCredentialResponse, openScreen: (String) -> Unit) {
+        when (val credential = result.credential) {
+            is CustomCredential -> {
+                if (credential.type == TYPE_GOOGLE_ID_TOKEN_CREDENTIAL) {
+                    linkAccountWithGoogle(credential, openScreen)
+                } else {
+                    Log.d(ERROR_TAG, UNEXPECTED_CUSTOM_CREDENTIAL)
+                }
+            }
+            else -> {
+                Log.d(ERROR_TAG, UNEXPECTED_CREDENTIAL)
```

Comment:
Seems like these are error logs. Should we use `Log.e()` instead?

Reply:
Good catch! Fixed.